fury/actors/tensor.py

```diff
@@ -0,0 +1,213 @@
+import os
+
+import numpy as np
+
+from fury import actor
+from fury.shaders import (attribute_to_actor, import_fury_shader,
+                          shader_to_actor, compose_shader)
+
+
+def tensor_ellipsoid(centers, axes, lengths, colors, scales, opacity):
+    """
+    Visualize one or many Tensor Ellipsoids with different features.
+
+    Parameters
+    ----------
+    centers : ndarray(N, 3)
+        Tensor ellipsoid positions
+    axes : ndarray (3, 3) or (N, 3, 3)
```

Comment:
Does it have to be between [0-1] ? if yes, it would be good to use `np.clip` in case there is crazy values